src/Workspaces/Core/Portable/Remote/WellKnownServiceHubServices.cs

```diff
@@ -6,24 +6,26 @@ namespace Microsoft.CodeAnalysis.Remote
 {
     internal static class WellKnownServiceHubServices
     {
+        public const string NamePrefix = "roslyn";
+
         public static void Set64bit(bool x64)
         {
             var bit = x64 ? "64" : "";
 
-            SnapshotService = "roslynSnapshot" + bit;
-            CodeAnalysisService = "roslynCodeAnalysis" + bit;
-            RemoteDesignerAttributeService = "roslynRemoteDesignerAttributeService" + bit;
-            RemoteProjectTelemetryService = "roslynRemoteProjectTelemetryService" + bit;
-            RemoteSymbolSearchUpdateEngine = "roslynRemoteSymbolSearchUpdateEngine" + bit;
-            LanguageServer = "roslynLanguageServer" + bit;
+            SnapshotService = NamePrefix + "Snapshot" + bit;
+            CodeAnalysisService = NamePrefix + "CodeAnalysis" + bit;
+            RemoteDesignerAttributeService = NamePrefix + "RemoteDesignerAttributeService" + bit;
+            RemoteProjectTelemetryService = NamePrefix + "RemoteProjectTelemetryService" + bit;
+            RemoteSymbolSearchUpdateEngine = NamePrefix + "RemoteSymbolSearchUpdateEngine" + bit;
+            LanguageServer = NamePrefix + "LanguageServer" + bit;
         }
 
-        public static string SnapshotService { get; private set; } = "roslynSnapshot";
-        public static string CodeAnalysisService { get; private set; } = "roslynCodeAnalysis";
-        public static string RemoteSymbolSearchUpdateEngine { get; private set; } = "roslynRemoteSymbolSearchUpdateEngine";
-        public static string RemoteDesignerAttributeService { get; private set; } = "roslynRemoteDesignerAttributeService";
-        public static string RemoteProjectTelemetryService { get; private set; } = "roslynRemoteProjectTelemetryService";
-        public static string LanguageServer { get; private set; } = "roslynLanguageServer";
+        public static string SnapshotService { get; private set; } = NamePrefix + "Snapshot";
```

Comment:
Does somebody else actually set these (given it's private set)? If so, does something else need to use the constant as well?

Reply:
I don't know what you mean. WatsonReporter.cs uses the NamePrefix. 
BTW, this entire file is needs refactoring, but that's for another time.